docs/pages/development/pointcloud_v1.rst

```diff
@@ -87,103 +87,135 @@ For example, this is the schema of a dataset using "PDRF 7":
     [
       {
         "name": "X",
-        "dataType": "float",
-        "size": 64
+        "dataType": "integer",
+        "size": 32
       },
       {
         "name": "Y",
-        "dataType": "float",
-        "size": 64
+        "dataType": "integer",
+        "size": 32
       },
       {
         "name": "Z",
-        "dataType": "float",
-        "size": 64
+        "dataType": "integer",
+        "size": 32
       },
       {
         "name": "Intensity",
         "dataType": "integer",
-        "size": 16
+        "size": 16,
+        "unsigned": true
```

Comment:
i'd personally prefer `signed` rather than `unsigned` because double negatives are confusing, but I don't feel strongly about it

Reply:
Not done - background: table schemas just have "signed" values but they are not marked as anything. Raster datasets added unsigned values which are now explicitly marked, so we ended up with a similar system to, say, C: integers are signed unless they are marked "unsigned".

Reply:
fair enough 👍